test_canonicaljson.py

```diff
@@ -24,11 +26,15 @@
     set_json_library,
 )
 
-from frozendict import frozendict
-
 import unittest
 from unittest import mock
 
+frozendict_type: Optional[Type]
+try:
+    from frozendict import frozendict as frozendict_type
+except ImportError:
+    frozendict_type = None
+
```

Comment:
Minor: I'd be tempted to do `from canonicaljson import frozendict_type` here

Reply:
Fixed in 7ddb3b8.